<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01" >
<html>

<head>
  <meta http-equiv="Content-Type" content="text/html; charset=gb2312">
  <title>Cube - help:xls</title>
  <link rel="stylesheet" href="../cube_style.css" type="text/css">
  <link rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.0.1/basic/jquery.qtip.min.css">
  <script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
  <script src="//cdn.jsdelivr.net/qtip2/2.0.1/basic/jquery.qtip.min.js"></script>
  <script>
    $(function(){$('[title]').qtip({hide:{delay:250,fixed:true},position:{at:'bottom left'}});});
  </script>
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-44819428-1', 'eopsf.org');
  ga('send', 'pageview');

</script>
</head>

<body>
<div class="container">
    <div class="mainmenu">
        <ul>
            <li><a href="http://cube.local/cube.html" >HOME</a></li>
            <li><a href="http://cube.local/help/help.html" >HELP</a></li>
            <li><a href="http://cube.local/code_dwnld.html">CODE</a></li>
            <li><a href="http://eopsf.org/">ABOUT US</a></li>
        </ul>
    </div>

    <div class="logo">
        <img src="../images/cube_banner.png" width="830">
    </div>

    <div class="main">
    
    <left_title>Spreadsheet output produced by Cube</left_title>

   <!------------------------------------------------------------------------->
   <table  border="0" cellpadding="0" cellspacing="0" width="90%" align="left" >

          <tr>
		<td>&nbsp;</td>
		<td>&nbsp;</td>
          </tr>

    

          <tr>
		<td align="left" colspan=2> <a href= "../conservation.html" name="cons">Conservation module</a>
		of Cube server produces spreadsheet (xls format) with  conservation score mapped onto the 
		reference sequence.
		</td>
          </tr> 

          <tr>
		<td align="left" colspan=2>

		<img src="../images/cons_xls.957px.png" width="957"> 
		</td>
          </tr> 
 
          <tr>
		<td align="left" colspan=2>  The column contents are the following:
                                         
		<ul> 
		<li> Position number in the alignment (if you did not provide the alignment yourself, you can find the 
                     alignment produced by Cube in the 'work directory' available for download on the results page).
		<li> Percentage of gaps at this alignment position. Large number indicates that many sequences 
                     in the alignment do not have this position at all. Note that the missing residues 
                      might be an artifact of protein sequencing 
                     or prediction from genomics studies. Ideally, you should replace such sequences with more reliable ones.
		     Otherwise, just take this column as a potential trouble sign.

		<li> Conservation score. The scoring method used can be changed at the input page. By default it is 
                     real-valued trace. <a href="help.html#refs">See here</a> for the original references. All scores are 
                     turned into <a name="frac">"fractional coverage:"</a> the top fraction to which each residue 
		     belongs according  to the used score. Thus "0.14" means 
                     "this residue belongs to the top 14% conserved residues". 
		     <p>
		     Very important thing to keep in mind here. This estimate should always be concluded with the caveat clause
                     "... based on the set of sequences and their alignment provided for the analysis."  So - "this residue 
                      belongs to top 14% conserved residues in all metazoan proteins." But the same residue might as well
                      belong to the top 43% conserved in  vertebrate sequences, simply because these  43% of positions
                      are identical in all vertebrate versions of the protein.
		      <p>
		      The fractional coverage score is further binned into 20 bins, and colored according to the colorbar
                      sown in the last column in the table.

		<li> Residue type in the representative (reference)  sequence.
		<li> Sequential number in the representative (reference)  sequence.
		<li> If the structure is provided, the following three columns appear:
		     <ul>
                          <li> Residue type in the structure
                          <li> PDB residue identifier (mostly corresponding to the sequential number in the structure).
                          <li> Surface. The column contains the word "surface" if the solvent accessibility for this residue
                               is > 10&#8491;<sup>2</sup>, 
                               according to <a href="http://swift.cmbi.ru.nl/gv/dssp/">DSSP</a> program.
		     </ul>
		<li> If the annotation is provided at the input page, it will appear in the last contents column 
                      (the very last column we reserve for the colorbar).
		</ul>

		</td>
          </tr>

         <tr>  
		<td align="left" colspan=2> <a href= "../specialization.html" name="spec">Specialization  module</a>
              of Cube server produces spreadsheet (xls format) with  both conservation and specialization score mapped on the 
		reference sequence.		
		</td>
		
          </tr> 

  
          <tr>
		<td align="left" colspan=2>

		<img src="../images/spec_xls.960px.png" width="960"> 
		</td>
          </tr> 
          <tr>
		<td align="left" colspan=2>  The number of columns will depend on the number of groups in the analysis, and on
               whether the structure and/or annotation were provided, 
		but they will always be one of the following:
                                         
		<ul> 
		<li> Position number in the alignment (if you did not provide the alignment yourself, you can find the 
                     alignment produced by Cube in the 'work directory' available for download at the results page).
		<li> Percentage of gaps a this alignment position. Large number indicates that many sequences 
                     in the alignment do not have this position at all. Note that the missing residues 
                      might be an artifact of protein sequencing 
                     or prediction from genomics studies. Ideally, you should replace such sequences with more reliable ones.
		     Otherwise, just take this column as a potential trouble sign.
		<li> Conservation - in the specialization module it is always <a href="help.html#refs">Shannon entropy</a>.
		     Both conservation and  specificity scores are turned into fractional coverage, 
		     <a href="#frac"> see above </a>.
		<li> Specificity:

                    <ul>
			 <li> 'Discriminant' positions - positions that are well conserved within each group under consideration
		      but different across different groups (labeled as 'specificity' in the Pymol session).
			 <li> 'Determinants' for each group  you provided. These are positions that are 
                      relatively well conserved in the reference group,
                      and different then in the remaining groups (note that now we are not saying anything about the conservation
                      status in other groups. The position may be conserved or variable in the other groups, as long as the overlap
                       with the amino acid types in the reference group is small).
		    </ul>
		<li> Representative sequences: residue type and sequential number in each.
		<li> If the structure is provided, the following three columns appear:
		     <ul>
                          <li> Residue type in the structure
                          <li> PDB residue identifier (mostly corresponding to the sequential number in the structure).
                          <li> Surface. The column contains the word "surface" if the solvent accessibility for this residue
                               is > 10&#8491;<sup>2</sup>, 
                               according to <a href="http://swift.cmbi.ru.nl/gv/dssp/">DSSP</a> program.
		     </ul>
		<li> If the annotation is provided at the input page, it will appear in the last contents column 
                      (the very last columns we reserve for the two colorbars - one for conservation and one for specialization).


		</ul>

		</td>
          </tr>




    </table>	  



    </div>

    <div class="footer">
           By <a href="http://eopsf.org/">EPSF</a>, 
           a part of <a href="http://www.bii.a-star.edu.sg/research/bmad/bmad.php"> 
           Biomolecular Modeling and Design Division</a>, 
           <a href="http://www.bii.a-star.edu.sg">BII</a>, <a href="http://www.a-star.edu.sg">A*STAR</a>. Design by Harpy.
    </div>


</div>


</body>
</html>

