{% extends "base.html" %}
{% block content %}

<div class="main">

    <!----------------------->
    <table  border="0" cellpadding="0" cellspacing="0" width="90%" align="left" >

        <tr>
            <td align="left" colspan=2>
                <div class="boxed">
                    <P>
                        Cube is a server for comparative analysis of protein sequences.
                        A group of sequences can be compared to find regions they have in common
                        ('<a href="../conservation.html">conservation analysis</a>').

                        Alternatively, if  it is known that the  sequences can be divided into several  groups or classes,
                        such as two
                        <a href="../db/html/homologues.html">paralogous</a> families of proteins,  or proteins from two different
                        <a href="http://en.wikipedia.org/wiki/Kingdom_%28biology%29">kingdoms</a>,
                        they can be contrasted to
                        find regions in which they differ,
                        conferring  the specific function to each group
                        ('<a href="../specialization.html">specialization analysis</a>').
                        Follow <a href="on_comparative_analysis.html">this link</a> to read more about why-and-how of protein sequence comparison.


                    <P>
                        The purpose of such an exercise is, typically, to help in the rational design of single-site
                        mutational experiments, to estimate the impact of SNPs, or guide computational docking.
                        Specialization analysis, in particular,  may help in the search of separation-of-function mutants.

                    <P>
                        Below is the contents list for this help page. If you are new to the subject, consider starting
                        from worked examples, even if you do not  have a slightest intention of actually  working through them.
                </div>
        </tr>

        <!----------------------->
        <tr> <td colspan=2> <hr width="100%"> <td></tr>
        <tr>
            <td align="left">
                <div class="topNav" align="left">
                    <ul>
                        <li><a href = "#input">   INPUT     </a>
                        <li><a href = "#names">   SEQUENCE NAMES    </a>
                        <li><a href = "#output">  OUTPUT    </a>
                        <li><a href = "#errs">    SERVER ERRORS    </a>
                        <li><a href = "#work">    WORKED EXAMPLES</a>
                        <li><a href = "#more">    MORE HELP</a>
                        <li><a href = "#refs">    REFERENCES</a>
                        <li><a href = "#citing">    CITING CUBE</a>
                    </ul>
                </div>
            </td>
            <td width = "5%"></td>
        </tr>

        <!----------------------->
        <tr> <td colspan=2> <hr width="100%"> <td></tr>
        <tr>
            <td align="left" > <left_title>
                <a name = "input">INPUT</a>
            </left_title></td>
        </tr>
        <tr><td>&nbsp;</td></tr>


        <!----------------------->
        <tr>
            <td colspan=2 align="left">
                <P>
                    <a name="conservation"> The only required input on the <a href="../conservation.html">conservation</a> page

                        is a set of sequences in <a href=" fasta_example.txt">fasta</a> format. You can read more about the FASTA format on its <a href='http://en.wikipedia.org/wiki/FASTA_format'>Wikipedia page</a>.
                <P>
                    If you do not have  sequences
                    homologous to your protein,  you can
                    <a href="getting_seqs.html">read more</a> about finding them by clicking <a href="getting_seqs.html">here</a>.

                <P>
                    Optionally,  the sequences can be pre-aligned (the server accepts
                    <a href=" fasta_example.txt">fasta</a> and <a href="gcg_example.txt">msf</a>
                    formats). If your sequences are already aligned, please tick the "my sequences are aligned" checkbox, otherwise
                    the server will ignore your original alignment.

                <P>
                    The reference sequence can be specified. All scores will be shown mapped onto this sequence.
                    (The positions that do not exists in this sequence will be skipped.
                    This is to avoid one extraneosly long sequence in the
                    input making the output unreadable.) If no reference sequence is given, the first sequence in the uploaded file
                    will be used.

                <P>
                    In addition, the structure can be provided. Check <a href="getting_structure.html">here</a>
                    if you are not sure whether your protein has a known structure.


                <P>
                    The default scoring method can be changed.
                    The references for the implemented scoring methods can be found <a href = "#refs">below</a>.

                <P>
                    The users are invited to provide any  information that they already may have about the protein residues
                    (such as transmembrane regions, post-translational modifications sites, catalytic residues and similar),
                    numbered according to any sequence in the alignment. This information is added to the downloadable table,
                    alongside the conservation score, residue type, and surface accessibility information. Note: this information is,
                    of course, uploaded to the computer that Cube runs on. If you believe you  have something sensitive, you can
                    add it to the  spreadsheet after you download it.

            </td>
        </tr>

        <!----------------------->
        <tr>
            <td colspan=2 align="left">
                <P>
                    <a name="conservation">On the <a href="../conservation.html">specialization</a> page,
                        the input consists of sequences already divided into meaningful groups.
                        The groups can be arbitrary, but typically they are expected to represent
                        <a href="../db/html/homologues.html">paralogous</a>  families
                        of proteins in comparable taxonomical samples, or protein
                        <a href="../db/html/homologues.html">orthologues</a> divided into clearly distinct
                        <a href="http://en.wikipedia.org/wiki/Biological_classification">taxonomical groups</a>.


                <P>
                    Each group of sequences  should be in
                    <a href=" fasta_example.txt">fasta</a> format if the sequences are not aligned, or

                    <a href=" fasta_example.txt">fasta</a> or  <a href="gcg_example.txt">msf</a> if they are (aligned).

                <P>
                    Annotation, according to numbering in any of the sequences in the alignment can also be provided. It
                    will appear in  xls spreadsheet, alongside  the information about conservation/specialization and
                    surface accesibilty (if the structure was provided).

            </td>
        </tr>


        <!----------------------->
        <tr> <td colspan=2> <hr width="100%"> <td></tr>
        <tr>
            <td align="left" colspan=2> <left_title>
                <a name = "names">SEQUENCE NAMES </a>
            </left_title></td>
        </tr>
        <tr><td>&nbsp;</td></tr>

        <!----------------------->
        <tr >
            <td align="left" colspan=2>
                <P>
                    This is arguably a subtopic of the input section, but it is so important and potentially confusing,
                    that we believe it deserves a section of its own. In <a href=" fasta_example.txt">FASTA</a> format, in particular,
                    a sequence header might look something like this
                <pre>
           >gi|114647465|ref|XP_528667.2| PREDICTED: 4-hydroxyphenylpyruvate dioxygenase isoform 3 [Pan troglodytes]. 
               </pre>
                <P>
                    Note the '>' character - it is not arbitrary, nor optional. In FASTA format it marks the beginning of the header line.
                    (You can read more about the FASTA format on its <a href='http://en.wikipedia.org/wiki/FASTA_format'>Wikipedia page</a>).
                    In  this case, everything before the first space (e.g. 'gi|114647465|ref|XP_528667.2|')
                    will be taken as the input identifier for the sequence.
                    However, this identifier is a clunker, loaded with special characters, and completely un-informative regarding the
                    sequence it labels. Cube will therefore try to shorten the name to something like PAN_TRO_114647465.

                <P>
                    When adding the annotation, you can use 'gi|114647465|ref|XP_528667.2|' as the sequence name. Alternatively,
                    you can use any  unique  part of it - such as '114647465.'
                    The server will try to find the sequence you have in mind by pattern matching.
                    This  will fail if the selected name is not truly unique, and result in the input processing error.
                    Therefore, try and give unique names to your sequences. Also consider the possibilty that the seqeunces wiht
                    identical names actually
                    <i>are</i> two identical sequences - in that case you can get rid of one of them.

            </td>
        </tr>

        <!----------------------->
        <tr> <td colspan=2> <hr width="100%"> <td></tr>
        <tr>
            <td align="left" colspan=2> <left_title>
                <a name = "output">OUTPUT </a>
            </left_title></td>
        </tr>
        <tr><td>&nbsp;</td></tr>

        <!----------------------->
        <tr >
            <td align="left" colspan=2>
                <P>
                    <a href="conservation.html">Conservation.</a>
                    The server produces a 1D conservation map (the conservation score color coded and mapped on the sequence)
                    in the png format, the tabulated information (in xls format), and the conservation mapped onto the structure
                    (as a PyMol session). A consistent color coding is used in all three forms of the output.
            </td>
        </tr>

        <!----------------------->
        <tr>
            <td align="left"  colspan=2>
                <P>
                    <a href="specialization.html">Specialization.</a>
                    In the output, the specialization scores are shown side-by-side with the conservation values (Shannon entropy)
                    for each residue , both in the tabulated output (xls spreadsheet)
                    as well as  mapped on the structure (Pymol session).
                    The scores are also immediately shown in the browser, mapped on the sequence,
                    and in an html  version of the output table.
            </td>
        </tr>

        <!----------------------->
        <tr>
            <td align="left"  colspan=2>
                <P> Read more about the <a href="pymol_output.html">Pymol session</a> produced by Cube.

            </td>
        </tr>

        <!----------------------->
        <tr>
            <td align="left"  colspan=2>
                <P> Read more about the <a href="xls_output.html">xls spreadsheet</a> produced by Cube.

            </td>
        </tr>

        <!----------------------->
        <tr>
            <td align="left"  colspan=2>
                <P> If you are really interested in the inner workings of Cube, or if you would
                    simply like to download everything with a single click, Cube offers the whole
                    work directory for download. <a href="workdir_output.html">Read more</a>.

            </td>
        </tr>

        <!----------------------->
        <tr>
            <td align="left"  colspan=2>
                <P> <a name = "exons">Exon boundaries</a>. If the exons boundaries are known, you can indicate them in your reference sequence by inserting 'Z' character in their place.
                    Also if you use <a href="http://exolocator.eopsf.org/">ExoLocator server</a> to create the input sequence alignment, it will provide that piece of annotation for you.
                    In the conservation map, the exon boundaries are indicated as blue bars.

            </td>
        </tr>

        <!----------------------->
        <tr> <td colspan=2> <hr width="100%"> <td></tr>
        <tr>
            <td align="left" > <left_title>
                <a name = "errs"> SERVER ERRORS</a>
            </left_title></td>
        </tr>
        <tr><td>&nbsp;</td></tr>

        <!----------------------->
        <tr>
            <td align="left" colspan=2>
                <P>
                    We are doing our best to keep the server as robust as possible against all sorts of formatting that
                    crops in bio databases around the world. Sometimes the server fails, nevertheless. If it happens,
                    please send us the error message you received to
                    <a href="mailto:eopsfmail@gmail.com">our contact address</a>.

                <P>
                    In the meantime you can try to decipher the error message yourself
                    (like "Please provide the input sequences"), and perhaps see if simplifying the input format helps.
            </td>
        </tr>

        <!----------------------->
        <tr> <td colspan=2> <hr width="100%"> <td></tr>
        <tr>
            <td align="left" > <left_title>
                <a name = "work"> WORKED EXAMPLES</a>
            </left_title></td>
        </tr>
        <tr><td>&nbsp;</td></tr>

        <!----------------------->
        <tr>
            <td align="left" colspan=2>
                <P>
                    <a href="worked_examples/cons_ex1.html">
                        Conservation in HPPD Enzyme.</a>
                    Learn how to map  conservation onto protein sequence.
                    Create excel spreadheets with conservation data, and conservation maps in png format.
            </td>
        </tr>


        <tr>
            <td align="left" colspan=2>
                <P>
                    <a href="worked_examples/cons_ex2.html">
                        Conservation in HPPD Enzyme Redux.</a>
                    Learn how to map  conservation onto protein structure. Add annotation to the spreadsheet.
            </td>
        </tr>


        <tr>
            <td align="left" colspan=2>
                <P>
                    <a href="worked_examples/spec_ex1.html">Specialization in lactalbumin.</a>
                    After gene duplication 300 to 400 million years ago,
                    an enzyme lysozyme C gave rise to a gene that currently codes for Î±-lactalbumin,
                    a regulatory protein expressed only in the lactating mammary gland.
                    Learn how to map the specialization info onto protein sequence and structure.
                    Create excel spreadheets with specialization data, and specialization maps in png format.

            </td>
        </tr>

        <!-------
            <tr>
              <td align="left" colspan=2>
                       <P>
                <a href="worked_examples/spec_ex2.html">Difference between animal, fungal, and bacterial actin.</a>
                Use Cube to highlight protein specialization between
                <a href="http://en.wikipedia.org/wiki/Taxon">taxa</a> and
                <a href="http://en.wikipedia.org/wiki/Kingdom_%28biology%29">kingdoms</a>.


              </td>
        </tr>
           ------->



        <!----------------------->
        <tr> <td colspan=2> <hr width="100%"> <td></tr>
        <tr>
            <td align="left" > <left_title>
                <a name = "more">MORE HELP </a>
            </left_title></td>
        </tr>
        <tr><td>&nbsp;</td></tr>

        <tr>
            <td  align="left" > <a href="getting_seqs.html"> finding sequences </a>
            </td>
            <td  align="left" width="60%"> <a href="why_get_seqs.html">why find sequences on your own</a>
            </td>
        </tr>

        <tr>
            <td  align="left" width="60%"> <a href="getting_groups_of_seqs.html">does my protein have paralogues?</a>
            </td>
            <td  align="left" width="60%"> <a href="which_species.html">which species did my sequence come from?</a>
            </td>

        </tr>


        <tr>
            <td align="left" width="60%"> <a href="aligning_seqs.html">making an alignment and aligning alignments</a>
            </td>
            <td align="left"> <a href="viewing_alignment.html">viewing an  alignment</a>
            </td>
        </tr>



        <tr>
            <td align="left">  <a href="getting_structure.html">finding structure</a>
            </td>
            <td align="left"> <a href="viewing_structure.html">viewing structure</a>
        </tr>

        <tr>
            <td align="left"> <a href="getting_annotation.html">finding annotation</a>
            </td>
            <td align="left"> <a href="checklist.html">generic analysis checklist</a>
            </td>

        </tr>


        <tr>
            <td align="left"> <a href="http://cube.local/db/html/homologues.html">what's this orthologue/paralogue business?</a>
            </td>
            <td align="left">
            </td>

        </tr>



        <tr><td>&nbsp;</td></tr>



        <!----------------------->
        <tr> <td colspan=2> <hr width="100%"> <td></tr>
        <tr>
            <td align="left" > <left_title>
                <a name = "refs">REFERENCES </a>
            </left_title></td>
        </tr>
        <tr><td>&nbsp;</td></tr>

        <!----------------------->


        <tr>
            <td colspan=2 align="left">
                Conservation:
            </td>
        </tr>
        <tr>
            <td colspan=2 align="left">
                <ul>
                    <li>rvet: real-valued evolutionary trace Mihalek <em> et al.</em> (J Mol Biol. 2004, 336:1265,
                        <a href='http://www.ncbi.nlm.nih.gov/pubmed/?term=15037084'>PubMed</a>)

                    <li>entr: column entropy score based on Shannon's entropy; proposed independently by <br>
                        &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
                        Sander and Schneider
                        (1991, Proteins 9:56-68, <a href='http://www.ncbi.nlm.nih.gov/pubmed/?term=2017436'>PubMed</a>)
                        and  <br>
                        &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
                        Shenkin et al.(1991,Proteins 11:297-313, <a href='http://www.ncbi.nlm.nih.gov/pubmed/?term=1758884'>PubMed</a>)
                    <li>majority fraction:   Wu and Kabat (J Exp. Med, 1970 132(2):211-50,
                        <a href='http://www.ncbi.nlm.nih.gov/pubmed/?term=5508247'>PubMed</a>)
                    <li>valdar:  Valdar & Thornton
                        (2001, Proteins 42:108-124, <a href='http://www.ncbi.nlm.nih.gov/pubmed/?term=11093265'>PubMed</a>):
                    <li>integer-valued evolutionary trace: Lichtarge
                        (J. Mol. Biol. 1996, 257:342, <a href='http://www.ncbi.nlm.nih.gov/pubmed/?term=8609628'>PubMed</a>)

                </ul>
            </td>
        </tr>

        <tr>
            <td colspan=2 align="left">
                All methods with the name ending in "_s" have the amino acid alphabet size reduced from 20 to 9 by grouping residues into
                similarity groups. The similarity groups used are [V,I,L,A,M,G], [S,T], [D,E],  [K,R,H], [F,Y], and [Q,N].
                The remaining residue types are each in the group of their own.
            </td>
        </tr>


        <tr>
            <td colspan=2 align="left">
                &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
            </td>
        </tr>
        <tr>
            <td colspan=2 align="left">
                Specialization:
            </td>
        </tr>

        <tr>
            <td colspan=2 align="left">
                <ul>

                    <li>Zhang <em> et al.</em> ( <a href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0024382">
                        PloS one 6.9 (2011): e24382</a>, <a href='http://www.ncbi.nlm.nih.gov/pubmed/?term=21931701'>PubMed</a>)
                </ul>
            </td>
        <tr>

            <!----------------------->
        <tr> <td colspan=2> <hr width="100%"> <td></tr>
        <tr>
            <td align="left" > <left_title>
                <a name = "citing">CITING CUBE </a>
            </left_title></td>
        </tr>
        <tr><td>&nbsp;</td><td>&nbsp;</td></tr>
        <tr><td colspan=2 align="left">If you have found Cube useful in your work, please cite</td></tr>
        <tr><td colspan=2 align="left">Zhang, Zong Hong, Aik Aun Khoo, and Ivana Mihalek.
            "Cube-An Online Tool for Comparison and Contrasting of Protein Sequences."
            <a href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0079480">PloS one 8.11 (2013): e79480.e</a></td></tr>

        <tr><td>&nbsp;</td><td>&nbsp;</td></tr>


    </table>
    <!----------------------->

</div>

{% endblock content %}