{% extends "base.html" %}
{% block content %}

<div class="main">


    <left_title>Why look for your own sequences</left_title>

    <P> &nbsp;

    <P>
        An important  FAQ: why should I go look for my own collection of sequences? After all, there are plenty of websites that
        will provide prêt-à-porter sets for me.

    <P>
        Use those collections, by any means, but with the grain of salt. They were all constructed automatically, and
        all automated algorithms or pipelines tend to fail now and then.

    <P>
        In all likelihood, you are currently the world's premier expert for the protein you are working on. You are the most qualified person to decide which sequences belong to your analysis.

    <P>
        Take a look at your sequences (<a href="viewing_alignment.html">read here</a> about viewing sequences):
    <ul>
        <li> Does your set contain alternative splices for the same gene from the same species? Choose the one
            you want to work with. That would typically be the canonical or the longest splice.
        <li>  Are the sequences  complete?  Try to avoid fragments, if you can.
        <li>  Are they really all orthologues, or did some paralogues sneak in?
        <li>  Are the sequences so distant, that it looks
            like you would do better job aligning them then the alignment program did?
            If there are few positions to fix,
            go ahead and do it in <a href="viewing_alignment.html#sea">Seaview</a>, for example.
                Otherwise drop the sequences which are too distant from your
                protein of interest - bad alignments only bring in the noise into your discussion.
    </ul>

    <p>
        <a href="getting_seqs.html">Click here to read about getting the sequences. </a>

</div>

{% endblock content %}