<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01" >
<html>


<!----------------------------------------------------------------->
<!----------------------------------------------------------------->
<!----------------------------------------------------------------->
<head>
    <!----------------------->
    <meta http-equiv="Content-Type" content="text/html; charset=gb2312">

    <!----------------------->
    <title>Cube: why and how</title>

    <!----------------------->
    <link rel="stylesheet" href="../cube_style.css" type="text/css">
    <link rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.0.1/basic/jquery.qtip.min.css">

    <!----------------------->
    <script type="text/javascript" src="//ajax.googleapis.com/ajax/libs/jquery/1.8.3/jquery.min.js"></script>
    <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.0.1/basic/jquery.qtip.min.js"></script>
    <script>$(function(){$('[title]').qtip({hide:{delay:250,fixed:true}});});</script>
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-44819428-1', 'eopsf.org');
  ga('send', 'pageview');

</script>
</head>
<!----------------------------------------------------------------->
<!----------------------------------------------------------------->
<!----------------------------------------------------------------->

<body>


<div class="container">

    <!----------------------->
    <div class="mainmenu">
        <ul>
            <li><a href="http://cube.local/cube.html" >HOME</a></li>
            <li><a href="http://cube.local/help/help.html" >HELP</a></li>
            <li><a href="http://cube.local/code_dwnld.html">CODE</a></li>
            <li><a href="http://eopsf.org/">ABOUT US</a></li>
        </ul>
    </div>

    <!----------------------->
    <div class="logo">
        <img src="../images/cube_banner.png" width="830">
    </div>



    <div class="main">
       <!----------------------->
       <left_title> Comparing and contrasting  protein sequences - why and how</left_title>
       <p> &nbsp;

       <p>
       <div class="boxed">
       <b> Summary:</b>
       <ul>
	 <li> Given several groups or classes of sequences, one can look for, three  types (roughly)  of 
	   evolutionary behavior for a single amino acid: 
	   conserved across all groups, conserved-but-different in each group, or of varying degree of 
	   variability in each group.
	 <li> Bioinformaticians publish their work as methods/algorithms and their implementations, servers, 
	   and databases of pre-calculated results. Each has a different purpose and target audience.
	 <li> <a href="http://cube.local/cube.html">Cube</a> is a server. It can be used to highlight 
	   each of the three types of evolutionary  behavior at each position in a peptide.
       </ul>
	</div>


	    
      <p>

<h3 class="likesectionHead"><a 
 id="x1-1000"></a>Evolutionary behavior of biological sequences and the practical value of its analysis.</h3>
<!--l. 65--><p class="noindent" >
Comparative analysis of DNA or protein sequences  relies on  an intuitively appealing  model of their evolution:
  It  starts as a random process in which every region has an equal a priori chance of mutating. 
 However,   mutations that negatively impact a functionally important  region get cleared out of the population. 


<!--l. 68--><p class="indent" >   The mechanistic model in which functional positions on a protein sequence 
can acommodate only a small number of amino acid
types found one of its nicest confirmations in the series of experiments lead by
J.H. Miller <span class="cite">&#x00A0;[<a 
href="#Xmiller1977genetic">1</a>,&#x00A0;<a 
href="#Xsuckow1996genetic">2</a>]</span>. In this amazing quest, spanning over two decades, 
the impact of mutations in almost all  positions on
the DNA sequence encoding the bacterial protein LacI was examined. In the final paper of the series, Miller <i>et al </i>
were able to explain the consequence of almost all mutations resulting in a phenotype different
from the wild-type, as belonging to one of the several functional parts this protein possesses: folding
core, hinge, inducer binding pocket, and regions responsible for dimerization and interaction with
DNA.
<!--l. 73--><p class="indent" >   Evolution will thus restrict the number of residue types observable at each position to the set which is
allowable by the function. Nowadays, we can reproduce and trace the process in the lab <span class="cite">&#x00A0;
[<a href="#Xrobins2013coupling">3</a>]</span>. When
we analyze conservation of residues or nucleotides, we are reverse engineering a nature-devised system,
and looking for a plausible functional explanation for why particular residues are conserved
<span class="cite">&#x00A0;[<a href="#Xadikesavan2011separation">4</a>]</span>.


<p>
Then, noting that a prominent mechanism of genome evolution is gene duplication, we may enquire which
of the copies (termed paralogues) changes to acquire new function <span class="cite">&#x00A0;[<a 
href="#Xtaylor2004duplication">5</a>]</span>.  We can  look for  residues
that distinguish otherwise similar groups of genes or proteins. These may, but do not
need to be conserved in both paralogous groups <span class="cite">&#x00A0;[<a href="#Xgu1999statistical">6</a>]</span>.
  
After the gene duplication, the rate of evolution may stay the same in the two
newly-founded branches (homotachy, in the fanciful terminology of <span class="cite">&#x00A0;[<a 
href="#Xlopez2002heterotachy">8</a>]</span>, or type II divergence <span class="cite">&#x00A0;[<a 
href="#Xgu1999statistical">6</a>]</span>),
but is in general free to proceed at different rates (heterotachy, type I divergence). As a limiting
case of the former, a position may be conserved as a different residue type in each of the branches
(&#8221;constant-but-different&#8221; <span class="cite">&#x00A0;[<a 
href="#Xgribaldo2003functional">9</a>]</span>, discriminant <span class="cite">&#x00A0;[<a 
href="#Xbharatham2011determinants">10</a>]</span>), or even, as a further extreme, conserved across two groups of
related proteins.

In any case, locating positions with markedly different evolutionary behavior in different paralogues can 
be used to understand or  inform  redesign protein function <span class="cite">&#x00A0;[<a href="#Xkhoury2009computational">7</a>]</span>. 





<!--l. 77--><p class="indent" >   
There are several practical problems to solve, though,  to get meaningful results out of sequence comparison.

<p>
First, let us focus on the word "conserved."  We might notice
that it carries a hidden catch: it makes sense only when coupled with the definition of the set (or class) of sequences to
which it applies. (Conserved in all protein kinases or conserved in CK1 group? Conserved in all vertebrates, or in
mammals only?). The problem is twofold: we have to decide what defines the class
 of  sequences within which we want to look for the conservation, 
and, then, we need to find those  and only those sequences that belong to the class that we want to study.



<p class="indent" >   While  patterns of conservation or specialization are not hard to appreciate once they are pointed out,
they might be difficult to detect by a human observer - the alignment of one hundred vertebrate
genes can easily approach a
megabyte of data. Therefore, we would like to have ways to detect and classify of evolutionary behavior
computationally.




<!--l. 85--><p class="noindent" >
   <h3 class="likesectionHead"><a 
 id="x1-2000"></a>Methods and their implementations; servers and databases of pre-calculated results.</h3>
                                                                                       

                                                                                       
<!--l. 85--><p class="noindent" >When bioinformaticians develop methods for detecting any particular type of evolutionary behavior, the
fundamental way in which they present their work is by publishing the method - the scoring function or the
algorithm. It is the compact way, usually involving some algebra, for explaining what the method
does. At this point the methods may remain nameless. The names get attached later in the process
- to the <a href="http://en.wikipedia.org/wiki/Implementation#Computer_Science">implementations</a>,
 and even more often, to the servers. Sometimes  authors will offer
the implementation for download - if well written, this is the ultimate piece of documentation for a
method.
<!--l. 87--><p class="indent" >   However, using an implementation directly is a task for aficionados. Servers provide shortcuts for a broader
audience - they hide the implementational details from the user, and sometimes combine several sources of
information. They differ widely in the way they present the output - from plain text tables that
appear in the browser, to automatically generated printable reports and embedded visualization
tools. It is notable however that the value expected to be added by the server increases as the field
matures.
<!--l. 89--><p class="indent" >   Sometimes the involved pipeline is so complicated, prone to breaking down, difficult to completely automate,
or just time-consuming to complete, that the authors decide to present their results in the form of
databases of pre-calculated results. The drawback to databases is that their content is fixed, and
they do not allow the interested user to inquire how a change in the input data affects the offered
conclusions.
<!--l. 91--><p class="indent" >   Some of the methods discussed here can be found implemented in the omnibus annotation and
visualization applications <span class="cite">&#x00A0;[<a 
href="#Xporollo2007versatile">11</a>,&#x00A0;<a 
href="#Xschneider2012protein">12</a>]</span>, but the discussion of these is beyond the scope of this introductory
overview.
<!--l. 93--><p class="indent" >   Thus, papers by Valdar <span class="cite">&#x00A0;[<a 
href="#Xvaldar2001conservation">13</a>,&#x00A0;<a 
href="#Xvaldar">14</a>]</span> describe the eponymous score, 
<span class="cite">&#x00A0;[<a href="#Xrate4site">15</a>]</span> describes the phylogenetic method
implemented in rate4site program, <span class="cite">&#x00A0;[<a 
href="#Xlichtarge1997identification">16</a>]</span> and <span class="cite">&#x00A0;[<a 
href="#Xrvet">17</a>]</span> integer- and real-valued evolutionary trace respectively, and
<span class="cite">&#x00A0;[<a 
href="#Xbharatham2011determinants">10</a>]</span> the score used for detection of functional specialization in Cube. Diverge <span class="cite">&#x00A0;[<a 
href="#Xgu2002diverge">18</a>,&#x00A0;<a 
href="#Xgu2013update">19</a>]</span>, is the name of
the implementation of a statistical model of functional divergence described in <span class="cite">&#x00A0;[<a 
href="#Xgu1999statistical">6</a>,&#x00A0;<a 
href="#Xgu2001maximum">20</a>]</span>, and so
on.
<!--l. 95--><p class="indent" >   
<a href="http://www.ebi.ac.uk/thornton-srv/databases/cgi-bin/valdar/scorecons_server.pl">
The Scorecons Server</a>
provides access to the methods described in <span class="cite">&#x00A0;[<a 
href="#Xvaldar">14</a>]</span> (with the implementation apparenlty not
available). SPEER score <span class="cite">&#x00A0;[<a 
href="#Xchakrabarti2007functional">21</a>]</span>, is available both as a standalone version and a server <span class="cite">&#x00A0;[<a 
href="#Xchakraborty2012speer">22</a>]</span>, while
ConSurf<span class="cite">&#x00A0;[<a 
href="#Xarmon2001consurf">23</a>,&#x00A0;<a 
href="#Xlandau2005consurf">24</a>]</span> server combines the structural knowledge with the conservation estimates provided by
rate4site.
<!--l. 98--><p class="indent" >   As an example of authors deciding to present their results as a database, we refer the reader to the
ConSurf-DB <span class="cite">&#x00A0;[<a 
href="#Xgoldenberg2009consurf">25</a>]</span>, serving the purpose for the ConSurf pipeline, ET report maker <span class="cite">&#x00A0;[<a 
href="#Xmihalek2006evolutionary">26</a>]</span> for real-valued trace,
FunShift<span class="cite">&#x00A0;[<a 
href="#Xabhiman2005funshift">27</a>]</span> for the method presented in <span class="cite">&#x00A0;[<a 
href="#Xknudsen2001likelihood">28</a>]</span> or Cube-DB <span class="cite">&#x00A0;[<a 
href="#Xzhang2012cube">29</a>]</span> for <span class="cite">&#x00A0;[<a 
href="#Xbharatham2011determinants">10</a>]</span>.
<!--l. 102--><p class="indent" >   In terms of the underlying evolutionary events they aim to capture, the available applications can be
classified in broad strokes as follows.
<!--l. 104--><p class="indent" >   AMAS <span class="cite">&#x00A0;[<a 
href="#Xlivingstone1993protein">30</a>]</span> and evolutionary trace in its various incarnations <span class="cite">&#x00A0;[<a 
href="#Xivet">31</a>,&#x00A0;<a 
href="#Xinnis2000evolutionary">32</a>,&#x00A0;<a 
href="#Xrvet">17</a>,&#x00A0;<a 
href="#Xmihalek2006evolutionary">26</a>,&#x00A0;<a 
href="#Xmorgan2006viewer">33</a>,&#x00A0;<a 
href="#Xlua2010pyetv">34</a>]</span>, seek out positions
that are mutating slowly, according to the evolutionary scenario provided by the sequence similarity
tree. Rate4site uses a more advanced maximum-likelihood calculation to achieve the same, with
the application more difficult to scale to large sequence alignments. It can be found implemented
and enhanced by structural information in ConSurf<span class="cite">&#x00A0;[<a 
href="#Xarmon2001consurf">23</a>,&#x00A0;<a 
href="#Xlandau2005consurf">24</a>,&#x00A0;<a 
href="#Xgoldenberg2009consurf">25</a>]</span> and ConSurfDB<span class="cite">&#x00A0;[<a 
href="#Xgoldenberg2009consurf">25</a>]</span>. Similarly,
INTREPID server <span class="cite">&#x00A0;[<a 
href="#Xsankararaman2009intrepid">35</a>]</span>, implements a method <span class="cite">&#x00A0;[<a 
href="#Xsankararaman2008intrepid">36</a>]</span> that captures heuristically the variation in
each subtree. Notably, it comes with a variation that makes it applicable for detection of type II
events
<!--l. 106--><p class="indent" >   It is worth registering that the most straightforward methods, such as Shannon entropy and majority fraction
<span class="cite">&#x00A0;[<a 
href="#Xvaldar">14</a>]</span> actually work perfectly well for the purpose, especially if a &#8217;clean&#8217; set of orthologues is provided as an
input.
                                                                                       

                                                                                       
<!--l. 109--><p class="indent" >   Type 1 divergence is the type of behavior that seems to have the thinnest coverage in the literature. FunShift
<span class="cite">&#x00A0;[<a 
href="#Xabhiman2005funshift">27</a>]</span> database, collects results calculated using a maximum likelihood method to establish the rate of mutation
across branches of a presumptive evolutionary tree <span class="cite">&#x00A0;[<a 
href="#Xknudsen2001likelihood">28</a>]</span>. Diverge software is one of the few applications that can
autonomously distinguish between type I and type II behavior <span class="cite">&#x00A0;[<a 
href="#Xgu1999statistical">6</a>,&#x00A0;<a 
href="#Xgu2001maximum">20</a>]</span>, rather than taking type II behavior as a
trait to look for.
<!--l. 115--><p class="indent" >   Type 2 divergence is a model behind SDP prediction method <span class="cite">&#x00A0;[<a 
href="#Xkalinina2004automated">37</a>]</span>, and SDR database <span class="cite">&#x00A0;[<a 
href="#Xdonald2009sdr">38</a>]</span>,
implementing <span class="cite">&#x00A0;[<a 
href="#Xdonald2005determining">39</a>]</span>. They highlight the positions of functional importance that are conserved in all
paralogues, a model that works well for detection of catalytic sites of enzymes <span class="cite">&#x00A0;[<a 
href="#Xsaikat10ensemble">40</a>,&#x00A0;<a 
href="#Xrausell2010protein">41</a>]</span>. Treedet <span class="cite">&#x00A0;[<a 
href="#Xcarro2006treedet">42</a>]</span>
implements two methods <span class="cite">&#x00A0;[<a 
href="#Xdel2003automatic">43</a>]</span> aimed at capturing this type of behavior, as does <span class="cite">&#x00A0;[<a 
href="#Xcapra2008characterization">44</a>]</span>, with the
implementation available from the authors. Multi-RELIEF method and server<span class="cite">&#x00A0;[<a 
href="#Xye2008multi">45</a>]</span> applies a machine learning
method (RELIEF) established in other informatics fields and applies it to this type of behavior.
The algorithm does not require the input classification of the sequences, but comes up with it on
its own. SPEER-server <span class="cite">&#x00A0;[<a 
href="#Xchakraborty2012speer">22</a>]</span>, with the methodology described in <span class="cite">&#x00A0;[<a 
href="#Xchakrabarti2007functional">21</a>]</span> uses rate4site to estimate
within-group conservation, and ultimately, looks for positions with low mutational rates in all provided
groups.



    <h3 class="likesectionHead"><a 
 id="x1-3000"></a>Why Cube.</h3>
<!--l. 125--><p class="noindent" >It should be noted in the light of the above discussion that Cube is not in itself  a method. Rather, It is a server,
providing access to  several methods. Behind the server are two pieces of code (available <a href="http://cube.local/code_dwnld.htmlhttp://cube.local/code_dwnld.html">here</a>) implementing several conservation<span class="cite">&#x00A0;[<a 
href="#Xvaldar">14</a>,&#x00A0;<a 
href="#Xivet">31</a>,&#x00A0;<a 
href="#Xrvet">17</a>]</span> and one specialization detection method <span class="cite">&#x00A0;[<a href="#Xbharatham2011determinants">10</a>]</span>. Its
sister database of pre-calculated results is called Cube-DB, and it can be found <a href="http://cube.local/db/html/home.html">here</a>. The database,
however, collects only the results for vertebrate proteins. The purpose of Cube server is to enable
conservation and specialization scoring for any selection of sequences provided by the user. These can be
vertebrate sequences outside of the scope of ENSEMBL, other animal, fungal, bacterial, or even viral
sequences.


<!--l. 128--><p class="indent" >   The specialization method implemented in Cube allows description of both divergence type I and type II
events. It is a lightweight application with the aim of presenting our work in a format that we have found to be
practical in development and planning of experiments (mutagenesis experiments in particular): tabulation, mapping on the
structure, and the sequence - the image that can further be annotated. It leaves the user fully in control over the
sequences that the analysis is based on. It furthermore places side-by-side and invites the contemplation of three
types of evolutionary behavior: conservation and type I and type II specialization, conserved vs. determinant and
discriminating residues.



   <h3 class="likesectionHead"><a 
 id="x1-3000"></a>References</h3>
<!--l. 1--><p class="noindent" >
     <div class="thebibliography">
     <p class="bibitem" ><span class="biblabel">
    1.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xmiller1977genetic"></a>Miller JH, Ganem D, Lu P, Schmitz A (1977) Genetic studies of the lac repressor: I. correlation
     of mutational sites with specific amino acid residues: Construction of a colinear gene-protein map.
     Journal of molecular biology 109: 275&#8211;298.
     </p>
     <p class="bibitem" ><span class="biblabel">
    2.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xsuckow1996genetic"></a>Suckow J, Markiewicz P, Kleina L, Miller J, Kisters-Woike B, et&#x00A0;al. (1996) Genetic studies of
     the lac repressor XV: 4000 single amino acid substitutions and analysis of the resulting phenotypes
     on the basis of the protein structure. J Mol Biol 261: 509-523.
     </p>
     <p class="bibitem" ><span class="biblabel">
    3.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xrobins2013coupling"></a>Robins  WP,  Faruque  SM,  Mekalanos  JJ  (2013)  Coupling  mutagenesis  and  parallel  deep
     sequencing to probe essential residues in a genome or gene. Proceedings of the National Academy
     of Sciences 110: E848&#8211;E857.
     </p>
     <p class="bibitem" ><span class="biblabel">
    4.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xadikesavan2011separation"></a>Adikesavan AK, Katsonis P, Marciano DC, Lua R, Herman C, et&#x00A0;al. (2011) Separation of
     recombination and sos response in escherichia coli reca suggests lexa interaction sites. PLoS genetics
     7: e1002244.
     </p>
     <p class="bibitem" ><span class="biblabel">
    5.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xtaylor2004duplication"></a>Taylor JS, Raes J (2004) Duplication and divergence: the evolution of new genes and old ideas.
     Annu Rev Genet 38: 615&#8211;643.
                                                                                       

                                                                                       
     </p>
     <p class="bibitem" ><span class="biblabel">
    6.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xgu1999statistical"></a>Gu X (1999) Statistical methods for testing functional divergence after gene duplication. Mol
     Biol Evol 16: 1664-1674.
     </p>
     <p class="bibitem" ><span class="biblabel">
    7.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xkhoury2009computational"></a>Khoury GA, Fazelinia H, Chin JW, Pantazes RJ, Cirino PC, et&#x00A0;al. (2009) Computational
     design of candida boidinii xylose reductase for altered cofactor specificity.  Protein Science 18:
     2125&#8211;2138.
     </p>
     <p class="bibitem" ><span class="biblabel">
    8.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xlopez2002heterotachy"></a>Lopez P, Casane D, Philippe H (2002) Heterotachy, an important process of protein evolution.
     Mol Biol Evol 19: 1-7.
     </p>
     <p class="bibitem" ><span class="biblabel">
    9.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xgribaldo2003functional"></a>Gribaldo  S,  Casane  D,  Lopez  P,  Philippe  H  (2003)  Functional  divergence  prediction  from
     evolutionary analysis: a case study of vertebrate hemoglobin. Mol Biol Evol 20: 1754-1759.
     </p>
     <p class="bibitem" ><span class="biblabel">
   10.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xbharatham2011determinants"></a>Bharatham K, Zhang ZH, Mihalek I (2011) Determinants, discriminants, conserved residues - a
     heuristic approach to detection of functional divergence in protein families. PLoS ONE 6: e24382.
     </p>
     <p class="bibitem" ><span class="biblabel">
   11.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xporollo2007versatile"></a>Porollo A, Meller J (2007) Versatile annotation and publication quality visualization of protein
     complexes using polyview-3d. BMC bioinformatics 8: 316.
     </p>
     <p class="bibitem" ><span class="biblabel">
   12.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xschneider2012protein"></a>Schneider  G,  Sherman  W,  Kuchibhatla  D,  Ooi  HS,  Sirota  FL,  et&#x00A0;al.  (2012)  Protein
     sequence&#8211;structure&#8211;function&#8211;network   links   discovered   with   the   annotator   software   suite:
     Application to elys/mel-28. In: Computational Medicine, Springer. pp. 111&#8211;143.
     </p>
     <p class="bibitem" ><span class="biblabel">
   13.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xvaldar2001conservation"></a>Valdar WS, Thornton JM (2001) Conservation helps to identify biologically relevant crystal
     contacts. Journal of molecular biology 313: 399&#8211;416.
     </p>
     <p class="bibitem" ><span class="biblabel">
   14.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xvaldar"></a>Valdar W (2002) Scoring residue conservation. Proteins 48: 227-241.
     </p>
     <p class="bibitem" ><span class="biblabel">
   15.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xrate4site"></a>Pupko T, Bell R, Mayrose I, Glaser F, Ben-Tal N (2002) Rate4Site: an algorithmic tool for the
     identification of functional regions in proteins. Bioinformatics 18: S71-77.
                                                                                       

                                                                                       
     </p>
     <p class="bibitem" ><span class="biblabel">
   16.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xlichtarge1997identification"></a>Lichtarge O, Yamamoto K, Cohen F (1997) Identification of functional surfaces of the zinc
     binding domains of intracellular receptors1. J Mol Biol 274: 325-337.
     </p>
     <p class="bibitem" ><span class="biblabel">
   17.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xrvet"></a>Mihalek I, Re&#353; I, Lichtarge O (2004) A family of evolution&#8211;entropy hybrid methods for ranking
     protein residues by importance. J Mol Biol 336: 1265-1282.
     </p>
     <p class="bibitem" ><span class="biblabel">
   18.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xgu2002diverge"></a>Gu X, Vander&#x00A0;Velden K (2002) DIVERGE: phylogeny-based analysis for functional-structural
     divergence of a protein family. Bioinformatics 18: 500-501.
     </p>
     <p class="bibitem" ><span class="biblabel">
   19.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xgu2013update"></a>Gu X, Zou Y, Su Z, Huang W, Zhou Z, et&#x00A0;al. (2013) An update of diverge software for functional
     divergence analysis of protein family. Molecular biology and evolution .
     </p>
     <p class="bibitem" ><span class="biblabel">
   20.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xgu2001maximum"></a>Gu  X  (2001)  Maximum-likelihood  approach  for  gene  family  evolution  under  functional
     divergence. Mol Biol Evol 18: 453-464.
     </p>
     <p class="bibitem" ><span class="biblabel">
   21.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xchakrabarti2007functional"></a>Chakrabarti S, Bryant S, Panchenko A (2007) Functional specificity lies within the properties
     and evolutionary changes of amino acids. J Mol Biol 373: 801-810.
     </p>
     <p class="bibitem" ><span class="biblabel">
   22.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xchakraborty2012speer"></a>Chakraborty A, Mandloi S, Lanczycki CJ, Panchenko AR, Chakrabarti S (2012) Speer-server:
     a web server for prediction of protein specificity determining sites.   Nucleic acids research 40:
     W242&#8211;W248.
     </p>
     <p class="bibitem" ><span class="biblabel">
   23.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xarmon2001consurf"></a>Armon A, Graur D, Ben-Tal N (2001) Consurf: an algorithmic tool for the identification of
     functional regions in proteins by surface mapping of phylogenetic information. Journal of molecular
     biology 307: 447&#8211;463.
     </p>
     <p class="bibitem" ><span class="biblabel">
   24.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xlandau2005consurf"></a>Landau  M,  Mayrose  I,  Rosenberg  Y,  Glaser  F,  Martz  E,  et&#x00A0;al.  (2005)  ConSurf  2005:  the
     projection of evolutionary conservation scores of residues on protein structures. Nucleic Acids Res
     33: W299-302.
                                                                                       

                                                                                       
     </p>
     <p class="bibitem" ><span class="biblabel">
   25.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xgoldenberg2009consurf"></a>Goldenberg O, Erez E, Nimrod G, Ben-Tal N (2009) The consurf-db: pre-calculated evolutionary
     conservation profiles of protein structures. Nucleic acids research 37: D323&#8211;D327.
     </p>
     <p class="bibitem" ><span class="biblabel">
   26.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xmihalek2006evolutionary"></a>Mihalek I, Re&#353; I, Lichtarge O (2006) Evolutionary and structural feedback on selection of
     sequences for comparative analysis of proteins. Proteins 63: 87-99.
     </p>
     <p class="bibitem" ><span class="biblabel">
   27.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xabhiman2005funshift"></a>Abhiman S, Sonnhammer E (2005) Funshift: a database of function shift analysis on protein
     subfamilies. Nucleic Acids Res 33: D197-200.
     </p>
     <p class="bibitem" ><span class="biblabel">
   28.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xknudsen2001likelihood"></a>Knudsen  B,  Miyamoto  M  (2001)  A  likelihood  ratio  test  for  evolutionary  rate  shifts  and
     functional divergence among proteins. Proc Natl Acad Sci USA 98: 14512-14517.
     </p>
     <p class="bibitem" ><span class="biblabel">
   29.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xzhang2012cube"></a>Zhang  ZH,  Bharatham  K,  Chee  SM,  Mihalek  I  (2012)  Cube-db:  detection  of  functional
     divergence in human protein families. Nucleic acids research 40: D490&#8211;D494.
     </p>
     <p class="bibitem" ><span class="biblabel">
   30.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xlivingstone1993protein"></a>Livingstone C, Barton G (1993) Protein sequence alignments: a strategy for the hierarchical
     analysis of residue conservation. Bioinformatics 9: 745-756.
     </p>
     <p class="bibitem" ><span class="biblabel">
   31.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xivet"></a>Lichtarge O, Yamamoto K, Cohen F (1997) Identification of functional surfaces of the zinc
     binding domains of intracellular receptors1. J Mol Biol 274: 325-337.
     </p>
     <p class="bibitem" ><span class="biblabel">
   32.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xinnis2000evolutionary"></a>Innis C, Shi J, Blundell T (2000) Evolutionary trace analysis of TGF-<span 
class="cmsy-10">{</span>beta<span 
class="cmsy-10">} </span>and related growth
     factors: implications for site-directed mutagenesis. Protein Eng 13: 839847.
     </p>
     <p class="bibitem" ><span class="biblabel">
   33.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xmorgan2006viewer"></a>Morgan DH, Kristensen DM, Mittelman D, Lichtarge O (2006) Et viewer: an application for
     predicting and visualizing functional sites in protein structures. Bioinformatics 22: 2049&#8211;2050.
     </p>
     <p class="bibitem" ><span class="biblabel">
   34.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xlua2010pyetv"></a>Lua RC, Lichtarge O (2010) Pyetv: a pymol evolutionary trace viewer to analyze functional
     site predictions in protein complexes. Bioinformatics 26: 2981&#8211;2982.
                                                                                       

                                                                                       
     </p>
     <p class="bibitem" ><span class="biblabel">
   35.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xsankararaman2009intrepid"></a>Sankararaman S, Kolaczkowski B, Sjölander K (2009) Intrepid: a web server for prediction of
     functionally important residues by evolutionary analysis. Nucleic acids research 37: W390&#8211;W395.
     </p>
     <p class="bibitem" ><span class="biblabel">
   36.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xsankararaman2008intrepid"></a>Sankararaman S, Sjolander K (2008) INTREPID&#8211;INformation-theoretic TREe traversal for
     Protein functional site IDentification. Bioinformatics 24: 2445-2452.
     </p>
     <p class="bibitem" ><span class="biblabel">
   37.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xkalinina2004automated"></a>Kalinina O, Mironov A, Gelfand M, Rakhmaninova A (2004) Automated selection of positions
     determining functional specificity of proteins by comparative analysis of orthologous groups in
     protein families. Protein Sci 13: 443-456.
     </p>
     <p class="bibitem" ><span class="biblabel">
   38.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xdonald2009sdr"></a>Donald J, Shakhnovich E (2009) Sdr: a database of predicted specificity-determining residues
     in proteins. Nucleic Acids Res 37: D191-194.
     </p>
     <p class="bibitem" ><span class="biblabel">
   39.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xdonald2005determining"></a>Donald JE, Shakhnovich EI (2005) Determining functional specificity from protein sequences.
     Bioinformatics 21: 2629&#8211;2635.
     </p>
     <p class="bibitem" ><span class="biblabel">
   40.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xsaikat10ensemble"></a>Chakrabarty S, Panchenko A (2010) Ensemble approach to predict specificity determinants:
     benchmarking and validation. BMC Bioinformatics 10.
     </p>
     <p class="bibitem" ><span class="biblabel">
   41.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xrausell2010protein"></a>Rausell A, Juan D, Pazos F, Valencia A (2010) Protein interactions and ligand binding: from
     protein subfamilies to functional specificity. Proc Natl Acad Sci USA 107: 1995-2000.
     </p>
     <p class="bibitem" ><span class="biblabel">
   42.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xcarro2006treedet"></a>Carro A, Tress M, De&#x00A0;Juan D, Pazos F, Lopez-Romero P, et&#x00A0;al. (2006) Treedet: a web server
     to explore sequence space. Nucleic Acids Res 34: W110-115.
     </p>
     <p class="bibitem" ><span class="biblabel">
   43.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xdel2003automatic"></a>del Sol&#x00A0;Mesa A, Pazos F, Valencia A (2003) Automatic methods for predicting functionally
     important residues. J Mol Biol 326: 1289-1302.
     </p>
     <p class="bibitem" ><span class="biblabel">
   44.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xcapra2008characterization"></a>Capra  J,  Singh  M  (2008)  Characterization  and  prediction  of  residues  determining  protein
     functional specificity. Bioinformatics 24: 1473-1480.
                                                                                       

                                                                                       
     </p>
     <p class="bibitem" ><span class="biblabel">
   45.<span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span><a 
 id="Xye2008multi"></a>Ye  K,  Anton&#x00A0;Feenstra  K,  Heringa  J,  Ijzerman  A,  Marchiori  E  (2008)  Multi-RELIEF:  a
     method to recognize specificity determining residues from multiple sequence alignments using a
     Machine-Learning approach for feature weighting. Bioinformatics 24: 18-25.
</p>
     </div>

    <div class="footer">
           By <a href="http://eopsf.org/">EPSF</a>, 
           a part of <a href="http://www.bii.a-star.edu.sg/research/bmad/bmad.php"> 
           Biomolecular Modeling and Design Division</a>, 
           <a href="http://www.bii.a-star.edu.sg">BII</a>, <a href="http://www.a-star.edu.sg">A*STAR</a>. Design by Harpy.
    </div>

</body></html> 

                                                                                       
	    
